composables/data/breakingNews.ts

```diff
@@ -12,9 +12,13 @@ export function normalizeFindBreakingNewsResponse(breakingNewsData: Record<strin
 }
 
 export function normalizeBreakingNews(newsItem: Record<string, any>): BreakingNews {
+  const naiveTagMatcher = /(<([^>]+)>)/ig // Good enough to clean up tags from a CMS rich text field
```

Comment:
naive or native?

Reply:
naive. as in it assumes straightforward simple html, so it works fine for data from our cms fields, but it doesn't cover all the edge cases it would need to like, properly sanitize arbitrary html.